src/platform/testing/e2e/form-tester/util.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const join = require('path').join;
+
+/**
+ * Grabs the all the data for a form.
+ * @typedef {Object} Rules
+ * @property {Stirng} extension - The file extension to look for
+ * @property {Array<String>} ignore - Specific filenames to ignore
+ * @property {Array<String>} only - Run tests on only these files
+ * ---
+ * @typedef {Object} DataSet
+ * @property {String} fileName - The file name
+ * @property {Object} contents - The parsed contents
+ * ---
+ * @param {String} path - The path it'll look in
+ * @param {Object} rules - The rules it'll follow to find the files
+ * @return {Array<DataSet>} - All the parsed files
+ */
+function getTestDataSets(path, rules = { extension: 'json' }) {
+  return fs
+    .readdirSync(path)
+    .filter(fileName => fileName.endsWith(rules.extension))
+    .filter(
+      fileName =>
+        Array.isArray(rules.only) ? rules.only.includes(fileName) : true,
```

Comment:
How would you use this? Just as a spot-check while you're working on a feature locally?